tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -516,6 +516,7 @@
   "DD_IAST_TELEMETRY_VERBOSITY": "iast_telemetry_verbosity",
   "DD_IAST_TRUNCATION_MAX_VALUE_LENGTH": "iast_truncation_max_value_length",
   "DD_IAST_DB_ROWS_TO_TAINT": "iast_db_rows_to_taint",
+  "DD_IAST_COOKIE_FILTER_PATTERN": "iast_cookie_filter_pattern",
```

Comment:
Don't forget to update dd-go too 🙂

Reply:
I'm waiting for the RFC approval, so I don't have to change the literal. But you are right